frame/run_all.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+import unittest
+suite = unittest.TestSuite()
+suite.addTest(unittest.TestLoader().discover('.'))
+runner = unittest.TextTestRunner()
+result = runner.run(suite)
+exit(0 if result.wasSuccessful() else 1)
```

Comment:
`python -m unittest` would be simpler?

Reply:
And, for just running the tests by hand, `xvfb-run python3 -m unittest` more convenient?

Reply:
Yep, this works. I'll drop the helper script